### PR TITLE
Feature #152 Add Leaderboards functionality to GameCetner

### PR DESCRIPTION
**Pull request by EugeneButusov**

### CODE REVIEW

File: Soomla/Assets/Plugins/Soomla/Profile/ProfileEvents.cs [hunk 1]
```diff
@@ -621,6 +770,18 @@ public void onInviteFailed(String message)
 
 		public static Action<Provider, string> OnInviteCancelled = delegate {};
 
+		public static Action<Provider, bool, string> OnGetLeaderboardsStarted = delegate {};
```
Comment (borisspektor) [inline Soomla/Assets/Plugins/Soomla/Profile/ProfileEvents.cs]: @EugeneButusov @Blinkq we'll need to add it to the branch with ValueObject as well

**Comment by EugeneButusov** [comment]
Sorry, wrong repo.

File: Soomla/Assets/Plugins/Soomla/Profile/ProfileEvents.cs (hunk 1 above)
Comment (Blinkq) [inline Soomla/Assets/Plugins/Soomla/Profile/ProfileEvents.cs]: @borisspektor  in our last conversation we didnt decide, should we merge ValueObject or not, so should we ?

Comment (borisspektor) [inline Soomla/Assets/Plugins/Soomla/Profile/ProfileEvents.cs]: @EugeneButusov @Blinkq I'll answer on correct PR :)

**Pull request closed without merging**